pgmpy/readwrite/ProbModelXML.py

```diff
@@ -358,6 +362,15 @@ def _add_constraint(self, constraint):
             value = constraint_data[name]
             etree.SubElement(constraint_element, 'Argument', attrib={'name': name, 'value': value})
 
+    def _add_decisioncriteria(self, criteria_dict):
```

Comment:
I recommend changing the function name to `_add_decision_criteria`, for better readability.